apps/server/src/modules/tldraw/service/tldraw.ws.service.ts

```diff
@@ -61,9 +55,15 @@ export class TldrawWsService {
 
 	public async closeConn(doc: WsSharedDocDo, ws: WebSocket): Promise<void> {
 		if (doc.connections.has(ws)) {
-			const controlledIds = doc.connections.get(ws) as Set<number>;
+			const controlledIds = doc.connections.get(ws);
+			let clientsArray: number[] = [];
+			if (controlledIds) {
+				clientsArray = Array.from(controlledIds);
+			}
+
 			doc.connections.delete(ws);
-			removeAwarenessStates(doc.awareness, Array.from(controlledIds), null);
+			removeAwarenessStates(doc.awareness, clientsArray, null);
```

Comment:
removeAwarenessStates(doc.awareness, **forceToArray(controlledIds)**, null);

Reply:
is that fine?